chat/message/src/Message/Message.stories.tsx

```diff
@@ -178,45 +178,49 @@ export const VerifiedAnswer: StoryObj<MessageStoryProps> = {
   },
 };
 
+const links = [
+  {
+    href: 'https://mongodb.design',
+    children: 'LeafyGreen UI',
+    variant: 'Website',
+  },
+  {
+    href: 'https://mongodb.github.io/leafygreen-ui/?path=/docs/overview-introduction--docs',
+    children: 'LeafyGreen UI Docs',
+    variant: 'Docs',
+  },
+  {
+    href: 'https://learn.mongodb.com/',
+    children: 'MongoDB University',
+    variant: 'Learn',
+  },
+  {
+    href: 'https://mongodb.com/docs',
+    children: 'MongoDB Docs',
+    variant: 'Docs',
+  },
+  {
+    href: 'https://www.youtube.com/watch?v=dQw4w9WgXcQ',
+    children: 'Rick Astley - Never Gonna Give You Up',
+    variant: 'Video',
+  },
+  {
+    href: 'https://mongodb.com/',
+    children: 'MongoDB Homepage',
+    variant: 'Website',
+  },
+];
+const getWithRichLinksChildren = () => (
+  <>
+    <MessageFeedback />
+    <MessageLinks links={links} />
+  </>
+);
 export const WithRichLinks: StoryObj<MessageStoryProps> = {
```

Comment:
nit: I know this was here before, but this story's name is a little misleading. The `How to Test Changes` section in the description says to "Navigate to MessageLinks in Storybook"  but there is no story with that name. Could this be changed to `withMessageLinks`?